sdk/storage/azure-storage-blob/azure/storage/blob/_list_blobs_helper.py

```diff
@@ -73,30 +133,29 @@ def _get_next_cb(self, continuation_token):
                 prefix=self.prefix,
                 marker=continuation_token or None,
                 maxresults=self.results_per_page,
-                cls=return_context_and_deserialized,
+                cls=deserialize_list_result,
```

Comment:
This is the `cls` parameter I mentioned that we would use to hook into the deserialization.